db/crud_test.go

```diff
@@ -827,20 +826,16 @@ func BenchmarkDatabaseGet1xRev(b *testing.B) {
 	defer tearDownTestDB(b, db)
 
 	body := Body{"foo": "bar", "rev": "1-a"}
-	_, _, err := db.PutExistingRevWithBody("doc1", body, []string{"1-a"}, false)
-	assert.NoError(b, err)
+	_, _, _ = db.PutExistingRevWithBody("doc1", body, []string{"1-a"}, false)
 
 	largeDoc := make([]byte, 1000000)
 	longBody := Body{"val": string(largeDoc), "rev": "1-a"}
-	_, _, err = db.PutExistingRevWithBody("doc2", longBody, []string{"1-a"}, false)
-	assert.NoError(b, err)
+	_, _, _ = db.PutExistingRevWithBody("doc2", longBody, []string{"1-a"}, false)
 
 	var shortWithAttachmentsDataBody Body
 	shortWithAttachmentsData := `{"test": true, "_attachments": {"hello.txt": {"data":"aGVsbG8gd29ybGQ="}}, "rev":"1-a"}`
-	err = base.JSONUnmarshal([]byte(shortWithAttachmentsData), &shortWithAttachmentsDataBody)
-	require.NoError(b, err)
-	_, _, err = db.PutExistingRevWithBody("doc3", shortWithAttachmentsDataBody, []string{"1-a"}, false)
-	assert.NoError(b, err)
+	base.JSONUnmarshal([]byte(shortWithAttachmentsData), &shortWithAttachmentsDataBody)
```

Comment:
explicitly ignore error to avoid errcheck flagging it

```suggestion
	_ = base.JSONUnmarshal([]byte(shortWithAttachmentsData), &shortWithAttachmentsDataBody)
```